packages/demo/frontend/src/components/home/GettingStarted.tsx

```diff
@@ -0,0 +1,5215 @@
+                        >{`marketBlocklist`}</span>
+                        {`: [
+      { `}
+                        <span
+                          style={{ color: colors.syntax.property }}
+                        >{`from`}</span>
+                        {`: `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`ETH`}</span>
+                        {`, `}
+                        <span
+                          style={{ color: colors.syntax.property }}
+                        >{`to`}</span>
+                        {`: `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`USDC`}</span>
+                        {` },
+      { `}
+                        <span
+                          style={{ color: colors.syntax.property }}
+                        >{`from`}</span>
+                        {`: `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`USDC`}</span>
+                        {`, `}
+                        <span
+                          style={{ color: colors.syntax.property }}
+                        >{`to`}</span>
+                        {`: `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`ETH`}</span>
+                        {` },
+    ],
+  },
+  `}
+                        <span
+                          style={{ color: colors.syntax.comment }}
+                        >{`// Chain Provider`}</span>
+                        {`
+  `}
+                        <span
+                          style={{ color: colors.syntax.property }}
+                        >{`chains`}</span>
+                        {`: [
+      `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`unichain`}</span>
+                        {`,
+      `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`optimism`}</span>
+                        {`,
+      `}
+                        <span
+                          style={{ color: colors.syntax.variable }}
+                        >{`base`}</span>
+                        {`
+  ]
+}`}
+                      </code>
+                    </pre>
+                    {/* Copy button */}
+                    <button
+                      onClick={() =>
+                        navigator.clipboard.writeText(
+                          `import { USDC, ETH, WBTC, USDT } from '@eth-optimism/actions-sdk/assets'
+import { ExampleMorphoMarket, ExampleAaveMarket } from '@eth-optimism/actions-sdk/markets'
+import { unichain, optimism, base } from 'viem/chains'
+import { getPrivyClient } from 'privy'
```

Comment:
nit: I found this a little confusing because there is no 'privy' library and no export for `getPrivyClient`. I think the correct import is technically:

```
import { PrivyClient } from '@privy-io/server-auth'
```

Reply:
done